.test-infra/jenkins/job_PerformanceTests_KafkaIO_IT.groovy

```diff
@@ -38,6 +38,19 @@ job(jobName) {
 
   (0..2).each { k8s.loadBalancerIP("outside-$it", "KAFKA_BROKER_$it") }
```

Comment:
You should be able to pass in additional parameters into the Kubernetes.create call since it invokes kubectl
See https://stackoverflow.com/questions/60449786/how-to-pass-command-line-argument-to-kubectl-create-command

Alternatively you might be able to let the loadbalancer choose the external port number and then get it via a `kubectl get services` or `kubectl describe services` command.